### PR TITLE
Azure CLI 2.0.76

### PR DESCRIPTION
**Pull request by azclibot**
The latest release of the Azure CLI

### CODE REVIEW

File: Formula/azure-cli.rb [hunk 1]
```diff
@@ -650,10 +645,6 @@ class AzureCli < Formula
   end
 
   def install
-    # Work around Xcode 11 clang bug
-    # https://code.videolan.org/videolan/libbluray/issues/20
-    ENV.append_to_cflags "-fno-stack-check" if DevelopmentTools.clang_build_version >= 1010
```
Comment (fxcoudert) [inline Formula/azure-cli.rb]: Is this removal intentional?

Comment (chenrui333) [reply Formula/azure-cli.rb]: I think it is, as that seems like a bot.

Comment (chenrui333) [reply Formula/azure-cli.rb]: cc @azclibot

Comment (roy-work) [reply Formula/azure-cli.rb]: @MyronFanQiu, who cut the 2.0.76 release commit, and works for Azure, and hopefully will either know the answer here, or knows who does. (And is human.)

(I'm just an onlooker who is interested in a feature in .76 and thus wants this PR.)

Comment (MyronFanQiu) [reply Formula/azure-cli.rb]: @fengzhou-msft Hi. Could you take a look?

Comment (fengzhou-msft) [reply Formula/azure-cli.rb]: @fxcoudert This removal is not intentional. We generate the formula by a template. I was not aware of this change here, so it's not in our template. Please add back here. I will add it in our template as well.

**Comment by fengzhou-msft** [comment]
@fxcoudert can you help add back the workaround for Xcode 11 and commit the changes? Thanks.

**Comment by fxcoudert** [comment]
If you can't add to this PR, let's close it. Either a new PR can be manually opened, or the bot retriggered (with the correct content), or we will wait for the next version.
